processing_service/insert_to_database.py

```diff
@@ -0,0 +1,166 @@
+from sqlalchemy import create_engine
+from sqlalchemy.dialects.postgresql import insert
+
+from configparser import ConfigParser
+from datetime import datetime as dt
+
+# Import database models
+import model.countries
+import model.organisations
+
+
+# Setting the current month
+current_month = dt.now().strftime('%B')
+
+# Temporary mapping between GBIF and GeoCASe
+organisation_mapping = {
```

Comment:
This list together with country_mapping is created once at the top of this file to solve the mapping between GBIF and GeoCASe because at the moment there are no common identifiers between the two

Reply:
Definitely a temporary solution